CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - The Medline/Pubmed search now also supports the [default fields and operators for searching](https://docs.jabref.org/collect/import-using-online-bibliographic-database#search-syntax). [forum#3554](https://discourse.jabref.org/t/native-pubmed-search/3354)
 - We improved group expansion arrow that prevent it from activating group when expanding or collapsing. [#7982](https://github.com/JabRef/jabref/issues/7982), [#3176](https://github.com/JabRef/jabref/issues/3176)
 - When configured SSL certificates changed, JabRef warns the user to restart to apply the configuration.
+- Increased the multi-instance network connection timeout to 1000ms.
```

Comment:
Not sure, but I think it'd be nice to mention the issue here, just to gibe a bit more context :)

Reply:
Changelogs should be understandable to the average user and be less technical: `We fixed an issue that caused JabRef to open multiple instances when "Remote Operation" is enabled` sounds good to me :-)